services/api/src/domain/game.ts

```diff
@@ -0,0 +1,15 @@
+export interface Turn {
+  row?: number;
+  tile?: number;
+  figure?: string;
+  player?: string;
```

Comment:
is this actually needed?

Reply:
Yes, it is. It is used to render player avatar(the letter) and player name in the board pop up